themes/default/content/blog/deploy-ai-ml-vercel-app/index.md

```diff
@@ -0,0 +1,204 @@
+
+Individual scripts or manual settings to set up and manage each component can soon create a tangled web of inefficiencies and potential problems. That's where Pulumi comes in as the pivotal figure in our AI/ML journey. From the frontend to the backend, Pulumi orchestrates the programmatic generation and coordination of every critical infrastructure element. It's the link that connects your lofty AI goals with the pragmatism of efficient development.
+
+With Pulumi, developer productivity increases, scalability is a breeze, and the complex infrastructure piece fits together with easily. Now that you can see why this is important, let's get started.
+
+Before we get into the details of using Pulumi to deploy a Vue.js frontend app to Vercel, let's start with some quick terminology/context:
+
+- [Katwalk Frontend](https://github.com/pulumiverse/katwalk/tree/main/src/app/katwalk-frontend) is a demo Vue.js application that integrates with the [Katwalk Server](https://github.com/pulumiverse/katwalk) to produce a function web interface for you to communicate with your chatbot. 
+- [Vercel](https://vercel.com/) is a modern web development cloud platform and hosting provider. It specialises in making web application deployment simple and efficient.
+
+## Integrate Your Chatbot Service With a Frontend And Deploy It!
+If you are ready to integrate your chatbot and deploy it to Vercel along with us, then let's double check a few requirements before we continue. While Pulumi supports many languages, we chose Python for this project, as Python is a familiar language in the AI industry and this is part of the *#MLOpsChallenge*.
+
+### Requirements
+
+* [Pulumi CLI](https://www.pulumi.com/docs/install/)
+* [Pulumi account and access token](https://app.pulumi.com/signup)
+* [Python3](https://www.python.org/downloads/)
+* [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* Endpoint of Katwalk Server API. Follow the steps in [this article](https://www.pulumi.com/blog/mlops-the-ai-challenge-is-cloud-not-code/) to deploy the API and get your endpoint.
+* [Vercel account and access token](https://vercel.com/guides/how-do-i-use-a-vercel-api-access-token#creating-an-access-token)
+
+
+Now that you have everything needed to get started, let's hook up our frontend and deploy the frontend to Vercel using Pulumi.
+
+### 1. Clone the repo and setup your project
+
+The first thing we need to do is clone the repository from GitHub and setup the Pulumi Python environment.
+
+```bash
+# Clone the repository and cd to the infra directory
+git clone https://github.com/pulumiverse/katwalk && cd katwalk/pulumi
```

Comment:
is this the right repo? I suspect you should be pointing them to your own repo that contains the frontend code right?